src/Illuminate/Validation/Concerns/FormatsMessages.php

```diff
@@ -426,18 +427,10 @@ protected function replaceIndexOrPositionPlaceholder($message, $attribute, $plac
      */
     protected function numberToIndexOrPositionWord(int $value)
     {
-        return [
-            1 => 'first',
-            2 => 'second',
-            3 => 'third',
-            4 => 'fourth',
-            5 => 'fifth',
-            6 => 'sixth',
-            7 => 'seventh',
-            8 => 'eighth',
-            9 => 'ninth',
-            10 => 'tenth',
-        ][(int) $value] ?? 'other';
+        return match ($value) {
+            1, 2, 3, 4, 5, 6, 7, 8, 9, 10 => Number::spelloutOrdinal($value, 'en'),
+            default => 'other',
+        };
```

Comment:
of course, this could also be this:
```suggestion
        return match (true) {
            $value >= 1 && $value <= 10 => Number::spelloutOrdinal($value, 'en'),
            default => 'other',
        };
```

Reply:
or this:
```suggestion
        if ($value >= 1 && $value <= 10) {
            return Number::spelloutOrdinal($value, 'en');
        }

        return 'other';
```